packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.styles.ts

```diff
@@ -299,12 +296,9 @@ export const getStyles = memoizeFunction(
           '.ms-ComboBox-Input': {
             color: palette.neutralDark,
             selectors: {
-              '::placeholder': {
-                color: palette.neutralPrimary
-              },
-              ':-ms-input-placeholder': {
-                color: palette.neutralPrimary
-              }
+              '::placeholder': hoverPlaceholderStyles, // Chrome, Safari, Opera, Firefox

```

Comment:
Can't you just do `::placeholder, :-ms-input-placeholder, ::-ms-input-placeholder` together?

Reply:
Quoting from [stack overflow](https://stackoverflow.com/questions/32096102/on-edge-browser-how-to-change-input-placeholder-text-color-using-css), "_Edge doesn't consinder ::placeholder a valid CSS and drops the entire rule._"

I also verified that having multiple directives on the same line does not work.